app/_src/production/mesh.md

```diff
@@ -100,6 +100,8 @@ kuma-dp run \
 {% endtab %}
 {% endtabs %}
 
+You can control which data plane proxies are allowed to join the mesh using [mesh constraints](/docs/{{ page.version }}/production/secure-deployment/dp-membership/).
```

Comment:
While reading this, it might be head scratching on how to achieve security with letting users to place annotations on deployments so I think it's useful to drop this link

Reply:
I think this will need to be backported because we only started single sourcing the Kuma docs in 2.1.x.

Reply:
Let's not bother backporting for net additions